helpers/ginutilext/extension.go

```diff
@@ -0,0 +1,63 @@
+package ginutilext
```

Comment:
I also have the this file in my local branch for wharf-provider-azuredevops right now.

My reasoning is that it makes it easier to move to wharf-core/pkg/ginutil, once they're more decided.

I couldn't find a good way to import a branch of a repository in Go.
If there is a way to do that then I could just make a branch for wharf-core and import that until the problems are ready to be pushed.

Not sure if this is a good way to do it, would love to here some other solutions.

Reply:
It seems like this might be solved by the merge order. If you cross link all the relevant PRs and number them by merge order. Then resolve this comment at the end just before this is ready for merge. If this isnt ready for merge yet because dependencies are missing you could switch this to a draft for now.